README.md

```diff
@@ -13,9 +13,23 @@ This repository contains **Dockerfile** of [oauth2_proxy](https://github.com/bit
 
 1. Install [Docker](https://www.docker.com/).
 
-2. Download [build](https://registry.hub.docker.com/u/crate/oauth2_proxy/) from public [Docker Hub Registry](https://registry.hub.docker.com/): `docker pull crate/oauth2_proxy`
+2. Download [build](https://registry.hub.docker.com/u/crate/oauth2_proxy/) from public [Docker Hub Registry](https://registry.hub.docker.com/):
 
-   (alternatively, you can build an image from Dockerfile: `docker build -t="crate/oauth2_proxy" github.com/crate/docker-oauth2_proxy`)
+```bash
+docker pull registry.cr8.net/crate/oauth2_proxy
+```
+
+Alternatively, you can build an image from Dockerfile:
+
+```bash
+docker build -t registry.cr8.net/crate/oauth2_proxy github.com/crate/docker-oauth2_proxy
+```
+
+Pushing the image to the registry:
+
+```bash
+docker push registry.cr8.net/crate/oauth2_proxy
```

Comment:
isn't a jenkins job doing that?

Reply:
No, there is no job for this package and I wouldn't add an automaged build for this kind of dockerfile-only packages.